app/scripts/templates/confirm_account_unlock.mustache

```diff
@@ -0,0 +1,22 @@
+<div id="main-content">
+  <header>
+    <h1 id="fxa-confirm-account-unlock-header">{{#t}}Unlock email sent{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success">
+      {{#t}}Email resent{{/t}}
+    </div>
```

Comment:
I'm not familiar with the CSS uses here - does this only become visible when the "resed email" action is invoked?